src/Symfony/Bundle/FrameworkBundle/Tests/Controller/TemplateControllerTest.php

```diff
@@ -32,13 +32,23 @@ public function testTwig()
         $this->assertEquals('bar', $controller('mytemplate')->getContent());
     }
 
-    public function testNoTwig()
+    public function testNoTwigViaTemplateAction()
     {
+        $controller = new TemplateController();
+
         $this->expectException(\LogicException::class);
         $this->expectExceptionMessage('You cannot use the TemplateController if the Twig Bundle is not available. Try running "composer require symfony/twig-bundle".');
-        $controller = new TemplateController();
 
         $controller->templateAction('mytemplate')->getContent();
+    }
+
+    public function testNoTwigViaInvokeMethod()
+    {
+        $controller = new TemplateController();
+
+        $this->expectException(\LogicException::class);
+        $this->expectExceptionMessage('You cannot use the TemplateController if the Twig Bundle is not available. Try running "composer require symfony/twig-bundle".');
+
         $controller('mytemplate')->getContent();
```

Comment:
I splitted the test in two tests, because both method, the `templateAction()` and `__invoke` throws the exception. I think we should test both behaviors independently